### PR TITLE
Update local team

### PR DESCRIPTION
**Pull request by cmaumet**

### CODE REVIEW

**Comment by cmaumet** [comment]
@carlosmalt - I've added the names of recruited team members for the local arrangements.

File: content/home/organizers.md [hunk 1]
```diff
@@ -26,7 +26,9 @@ TBD
 ### Local Arrangements
 Arnaud Legrand (CNRS)  
 Camille Maumet (INRIA)  
-  
+Matthieu Simonin (INRIA)
```
Comment (carlosmalt) [inline content/home/organizers.md]: Add a double blank after each of the new names to make them appear on their own line (a quirk of markdown).

**Review by carlosmalt — CHANGES_REQUESTED** [review]
Thank you @cmaumet -- Just need a small change before I can merge.

File: content/home/organizers.md (hunk 1 above)
Comment (cmaumet) [reply content/home/organizers.md]: Thankd @carlosmalt, that is now done ✔️

**Review by carlosmalt — APPROVED** [review]
Perfect!

**Pull request merged**